mne/viz/_3d.py

```diff
@@ -1490,3 +1628,217 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def _plot_dipole_mri_orthoview(dipole, trans, subject, subjects_dir=None,
+                               coord_frame='head', idx='gof', show_all=True,
+                               ax=None, block=False, show=True):
+    """Plot dipoles on top of MRI slices in 3-D.
+
+    Browse through the dipoles using mouse scroll or up/down arrows.
+
+    Parameters
+    ----------
+    dipole : instance of mne.Dipole
+        The dipole to plot.
```

Comment:
I would actually probably get rid of this docstring since these are all documented above.

But if you want to keep them, this is now out of date (can also be list)